src/App.js

```diff
@@ -3,22 +3,38 @@ import Home from "./views/Home";
 import CreateArticle from "./views/CreateArticle";
 import { useState, useEffect } from "react";
 
-function App({ prevArticles }) {
+function App() {
   const [articles, setArticles] = useState(
     JSON.parse(localStorage.getItem("articles")) || []
   );
 
+  const [deletedArticles, setDeletedArticles] = useState(
+    JSON.parse(localStorage.getItem("deletedArticles")) || []
+  );
+
   useEffect(() => {
     localStorage.setItem("articles", JSON.stringify(articles));
   }, [articles]);
 
+  useEffect(() => {
+    localStorage.setItem("deletedArticles", JSON.stringify(deletedArticles));
+  }, [deletedArticles]);
+
   return (
     <Router>
       <Route path="/" exact>
-        <Home articles={articles} setArticles={setArticles} />
+        <Home
+          articles={articles}
+          setArticles={setArticles}
+          setDeletedArticles={setDeletedArticles}
```

Comment:
I am not a big fan of passing the `setState` function down as props. It would be considered best practice to add a callback function. For now it is okay as it is 🙂